### ISSUE TITLE
Track quantity at Variant level

### ISSUE THREAD

**Issue by kjschaef** [etsy/etsyapi#109]
As a seller on Etsy, I desperately need software that integrates with Etsy to automatically track my inventory.  There are lots of options out there, but none of them work if the seller is using Listing Variations in Etsy.  They all say the Etsy API does not support quantity at the variant level.  Please consider adding this, as sellers currently are either forced to manually track inventory, or not use Listing Variations.

Thanks!

**Comment by FurnecoUk** [comment]
I completely agree with you. I'm a seller and all of my products have variations of one type or another. I don't know of many products on Etsy which don't have variations. The current inventory system on the Etsy website itself makes it impossible to control the stock levels of products with variations. It also makes it impossible to manage inventory levels from external software. Really struggling to develop external software without this feature. I agree, please consider adding this.

Thank you

**Comment by villanus** [comment]
You can easily track variation quantities in your own database, and then subtract them when an etsy order comes in (via the api),  This is how most 3rd party services do it.   The only trick is to map the variations properly using the API.

Etsy having variation quantities would probably benefit a small segment of sellers, while be confusing to others.   I am guessing this is why they never introduced it.

**Comment by mdg** [comment]
We are looking into adding this capability.

**Comment by villanus** [comment]
can you please let us know what happens to the v2 api once 3 is released?  i have a project i keep on the back bunrner beacuse i am worried v2 will be depreciated soon?  knowing etsy would keep v2 for a specific time after v3 is out wouldhelp ease theconcern
